### PR TITLE
Fixed tests

### PR DESCRIPTION
**Pull request by draganaHrcek**

### CODE REVIEW

File: web/src/test/java/eu/execom/testgenerator/web/controller/ReviewControllerTest.java [hunk 1]
```diff
@@ -98,52 +98,7 @@ public void testGetTestById_whenUserIsReviewerAndTestCompleted_expectOk() throws
...
-    //when
-    mockMvc.perform(
-      get(URL_PREFIX + "/1")
-          .header(AUTHENTICATION_TOKEN_KEY, token))
-    //then
-      .andExpect(status().isConflict());
-  }
-
-  @org.junit.Test
+ @org.junit.Test
```
Comment (stanislavgrujic) [inline web/src/test/java/eu/execom/testgenerator/web/controller/ReviewControllerTest.java:101]: Indentation was messed. Please add one more space.

File: web/src/test/java/eu/execom/testgenerator/web/integration/CategoryControllerIntegrationTest.java [hunk 2]
```diff
@@ -94,7 +96,7 @@ public void shouldGetAllCategories() throws Exception {
     mockMvc.perform(get(URL_PREFIX))
            .andExpect(status().isOk())
            .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8))
-           .andExpect(jsonPath("$.categoryResponses", hasSize(3)));
+           .andExpect(jsonPath("$.categoryResponses", hasSize(existingCategoriesSize+3)));
```
Comment (stanislavgrujic) [inline web/src/test/java/eu/execom/testgenerator/web/integration/CategoryControllerIntegrationTest.java:99]: Formatting.

File: web/src/test/java/eu/execom/testgenerator/web/integration/TestControllerIntegrationTest.java [hunk 3]
```diff
@@ -194,34 +194,17 @@ public void shouldDeleteTest() throws Exception {
     test.setStatus(TestStatus.REVIEW_FINISHED);
     test = testService.update(test);
     Long testId = test.getId();
+    int testsSize = (int) testService.findTestsForLoggedUserByEmail("name3@mailinator.com", null, 0).getTotalElements();
 
-    mockMvc.perform(get(URL_PREFIX).header(AUTHENTICATION_TOKEN_KEY, administratorToken)).andExpect(jsonPath("$.dashboardTestDtos", hasSize(1)));
+    mockMvc.perform(get(URL_PREFIX).header(AUTHENTICATION_TOKEN_KEY, administratorToken)).andExpect(jsonPath("$.dashboardTestDtos", hasSize(testsSize)));
     mockMvc.perform(delete(URL_PREFIX + "/" + testId).header(AUTHENTICATION_TOKEN_KEY, administratorToken)).andExpect(status().isNoContent());
-    mockMvc.perform(get(URL_PREFIX).header(AUTHENTICATION_TOKEN_KEY, administratorToken)).andExpect(jsonPath("$.dashboardTestDtos", hasSize(0)));
+    mockMvc.perform(get(URL_PREFIX).header(AUTHENTICATION_TOKEN_KEY, administratorToken)).andExpect(jsonPath("$.dashboardTestDtos", hasSize(testsSize-1)));
```
Comment (stanislavgrujic) [inline web/src/test/java/eu/execom/testgenerator/web/integration/TestControllerIntegrationTest.java:201]: Formatting

Comment (draganaHrcek) [reply web/src/test/java/eu/execom/testgenerator/web/integration/TestControllerIntegrationTest.java]: I applied formatter now. It made a lot of changes, I checked with Ivan and his formatter made same changes on class too. It seems like code was not formatted good before, just want to be sure that I didn't ruin format again. :)

**Pull request merged**